rfcs/0003-kubernetes-oci/README.md

```diff
@@ -134,6 +134,18 @@ spec:
       name: cosign-key
 ```
 
+### Layer selection
+
+By default, Flux assumes that the first layer of the OCI artifact contains the Kubernetes configuration.
+For multi-layer artifacts created by other tools than Flux CLI, users can specify the media type of the layer
+which contains the tarball with Kubernetes manifests.
```

Comment:
```suggestion
For multi-layer artifacts created by other tools than Flux CLI, users can specify the media type of the layer
which contains the tarball with Kubernetes manifests. When defined, the first layer matching this type will
be used.
```

To clarify how it behaves for ambiguous matches, as theoretically you can have multiple layers with the same media type.

Reply:
In practice I think that this will not suffice for a minority of projects which may add things in multiple layers, but I think that could be expanded in the future by adding more knobs to allow for composition.